docs/troubleshooting/different-private-key.mdx

```diff
@@ -0,0 +1,91 @@
+---
+title: Different Private Keys across Integrations
+image: "images/docs-meta-cards/documentation-card.png"
+displayed_sidebar: docs
+description: "Different Private Keys across Integrations | Documentation - Web3Auth"
+---
+
+Users frequently face confusion when they get different wallet addresses due to multitudes of reasons. Here's our attempt to explain the possible
+reasons for different wallet addresses and what to keep in mind while designing your solutions.
+
+## Factors affecting wallet addresses
+
+### Verifier
+
+A verifier is a piece of information about the OAuth provider being used by the application, which is queried by web3auth auth network nodes from a
+smart contract deployed on the Ethereum blockchain to verify the JWT token. To know more about creating verifiers, please refer
+[here](/docs/auth-provider-setup/verifiers).
+
+As the verifiers change, the private key changes. And as we go across different SDKs, we need different verifiers to be created, hence changing the
```

Comment:
This is incorrect. Tell them if they use the same verifier across every SDK their wallet addresses won't change. All SDKs support all verifiers

Reply:
You can say that across different login providers, the verifiers change, causing the change in keys even if the identifier, like sub, email etc remains the same. One can use Aggregate verifier in that case to keep the keys constant